lib/kit.js

```diff
@@ -28,6 +25,22 @@ class CoinsenceKit {
     return new CoinsenceKit(provider, signer, options);
   }
 
+  async init() {
+    return this.provider.getNetwork().then((network) => {
+      this.address = KitAddress[network.chainId.toString()];
+      
+      this.contract = new ethers.Contract(
+        this.address,
+        this.abi,
+        (this.signer || this.provider)
+      );
+
+      return this;
+    }).catch((error) => {
```

Comment:
I think we should not catch this error, but whoever calls this function should catch it. 
as this is a library we don't know if where it is used we actually want to log the error.

Reply:
okay